test/extended/openstack/loadbalancers.go

```diff
@@ -471,8 +471,15 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
 			g.By("Test that the canary service is accessible through new ingressController")
 			route, err := oc.AdminRouteClient().RouteV1().Routes("openshift-ingress-canary").Get(context.Background(), "canary", metav1.GetOptions{})
 			o.Expect(err).NotTo(o.HaveOccurred(), "canary route not found")
+			host := ""
+			for _, ingress := range route.Status.Ingress {
+				if strings.Contains(ingress.Host, domain) {
+					host = ingress.Host
+				}
+			}
```

Comment:
nit: Should we error out in case of `host` still being `""` after the iteration?

Reply:
done